miasm2/jitter/__init__.py

```diff
@@ -0,0 +1 @@
+"JIT feature"
```

Comment:
Maybe explain what it is meant for? I think it is more about emulating the real code than about just in time compilation in the end (I am thinking about the python jitter that does not really jit code if I remember well).
Maybe something like : "Code emulation and Just In Time (JIT) recompilation"?